### ISSUE TITLE
F# 7 Support

### ISSUE THREAD

**Issue by ntwilson** [ionide/ionide-vscode-fsharp#1802]
### Describe the bug
If I write any code that uses the new F# 7 SRTP features, the code works fine in FSI and builds fine with `dotnet build`, but shows with errors in Ionide.  

### Steps to reproduce
A quick repro is to make a script file with the code 
```f#
let inline f (a : 'a when 'a : (member X : int -> string)) = a.X 5
```
which will display an error in Ionide
> Lookup on object of indeterminate type based on information prior to this program point. A type annotation may be needed prior to this program point to constrain the type of the object. This may allow the lookup to be resolved.

## Machine info
 - OS: Windows 10
 - .NET SDK version: 7.0.100
 - Ionide version: 7.4.0

**Comment by baronfel** [comment]
Yep - we're blocked on supporting F# 7 language features until FCS is pushed out with a corrected MSBuild dependency. We're already working with the F# team to make this happen.

**Comment by ntwilson** [comment]
Feel free to close this, I just couldn't find where that work was discussed/tracked

**Comment by baronfel** [comment]
Nah, it's good to have this. Thanks for writing it up!

**Comment by dlidstrom** [comment]
@baronfel Is there an issue or PR where I can track the FCS progress?

**Comment by MikaelUmaN** [comment]
Wondering the same thing. It's a bit strange that F# VsCode development has to be blocked like this for a couple of months due to an incorrect dependency.

**Comment by baronfel** [comment]
Hey @MikaelUmaN! You can follow along the progress at here: https://github.com/fsharp/FsAutoComplete/pull/1043

**Comment by baronfel** [comment]
We released this in 7.5.0/7.5.1 today! Go forth and enjoy the new language features :)
